flask_bootstrap/templates/base/table.html

```diff
@@ -25,6 +25,8 @@
     {%- endwith %}
 {%- endmacro %}
 
+{% macro render_links(data) %}{% for part in data.strip().split(' ') %}{% if loop.index != 1 %} {% endif %}{{ part | urlize(target='_blank') }}{% endfor %}{%- endmacro %}
```

Comment:
Don't write all code in one line.

Why split the data?

Reply:
Because I did not yet know how to remove whitespace in the result. Now I do and have fixed this.